### PR TITLE
chore: update README

### LINKED ISSUE #170: Update README
### Context

Update repository README.

### PR DESCRIPTION
**Pull request by raducristianpopa**
## Context

Closes #170.

## Changes proposed in this pull request

To be merged after #184.

### CODE REVIEW

**Comment by raducristianpopa** [comment]
<h2>Extension builds preview</h2>

<table role="table">
<thead>
  <tr>
    <th>Name</th>
    <th>Link</th>
  </tr>
</thead>
<tbody>
  <tr>
    <td align="center">Latest commit</td>
    <td align="center">f0f0361404da43c3b0e075a12977d2959eacfed7</td>
  </tr>
  <tr>
    <td align="center">Latest job logs</td>
    <td align="center"><a href="https://github.com/interledger/web-monetization-extension/actions/runs/8297511091">Run #8297511091</a></td>
  </tr>
  <tr><td align="center">Badge</td><td align="center"><a href="https://github.com/interledger/web-monetization-extension/suites/21764055795/artifacts/1329675905">Download</a></td></tr><tr><td align="center">Badge</td><td align="center"><a href="https://github.com/interledger/web-monetization-extension/suites/21764055795/artifacts/1329675914">Download</a></td></tr><tr><td align="center">Badge</td><td align="center"><a href="https://github.com/interledger/web-monetization-extension/suites/21764055795/artifacts/1329675917">Download</a></td></tr><tr><td align="center">Badge</td><td align="center"><a href="https://github.com/interledger/web-monetization-extension/suites/21764055795/artifacts/1329675909">Download</a></td></tr>
</tbody>
</table>

**Review by ionutanin — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,103 +1,125 @@
-## Web Monetization Extension
+<h1 align="center">Web Monetization Extension</h1>
 
 ![Github Actions CI](https://github.com/interledger/web-monetization-extension/actions/workflows/sanity.yml/badge.svg?branch=main)
 [![Prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://prettier.io/)
 
+</center>
 Web Monetization is a browser extension that detects Web Monetization on websites using a browser API that allows the
```
Comment (tselit) [inline README.md:7]: Suggesting rewording: 
The Web Monetization browser extension is an open source implementation of the Web Monetization draft specification - learn more [here](https://webmonetization.org/specification/). The extension is built with React and TypeScript.

This Web Monetization browser extension: 
* Detects whether a user is focused on a website that is Web Monetization enabled. 
* Uses a browser API to create a payment stream from the authorized digital wallet or account of a user, to the website.

**Review by tselit — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,103 +1,125 @@
-## Web Monetization Extension
+<h1 align="center">Web Monetization Extension</h1>
```
Comment (huijing) [inline README.md:1]: did you want to do `<h1 text-align="center">Web Monetization Extension</h1>`?

File: README.md [hunk 3]
```diff
@@ -1,103 +1,125 @@
...
 
 ![Github Actions CI](https://github.com/interledger/web-monetization-extension/actions/workflows/sanity.yml/badge.svg?branch=main)
 [![Prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://prettier.io/)
 
+</center>
```
Comment (huijing) [inline README.md:6]: I think we don't need this.

**Review by tselit — APPROVED** [review]

**Pull request merged**